runtime/interpreter/statictype.go

```diff
@@ -95,19 +82,14 @@ func (t CompositeStaticType) String() string {
 	if t.Location == nil {
 		return t.QualifiedIdentifier
 	}
-	return string(t.TypeID)
```

Comment:
Changed to what is done for `InterfaceStaticType`

Reply:
In https://github.com/onflow/cadence/pull/2751 I improved composite and interface static types on master in a different way, by adding type ID caching to interface static types, instead of removing it from composite static types.

I'll revert the changes here once that PR is merged.